test/junit/scala/collection/StringParsersTest.scala

```diff
@@ -242,6 +242,7 @@ class StringParsersTest {
     "-0x0.000000000000090000000000000000001p-1022",
     "-0x0.00000000000009fffffffffffffffffffffffffffffffffp-1022",
     "-0x0.0000000000000fffffffffffffffffffffffffffffffffffp-1022",
+    ".4",
```

Comment:
Can we get "", "." and "4." too while we're at it?

Reply:
OK, I was too busy with "half" puns.

Reply:
Maybe change to "0.5" then 👍